SIT.Manager/ViewModels/Installation/ConfigureSitViewModel.cs

```diff
@@ -174,6 +182,39 @@ private void ValidateConfiguration()
             IsConfigurationValid = false;
             return;
         }
+
+        if (IsModsSelectionLoading || IsVersionSelectionLoading)
+        {
+            IsConfigurationValid = false;
+            return;
+        }
+    }
+
+    private async Task LoadAvailableModsList()
+    {
+        IsModsSelectionLoading = true;
+
+        await _modService.LoadMasterModList();
+        if (_modService.ModList.Count <= 1)
+        {
+            await _modService.DownloadModsCollection();
+            await _modService.LoadMasterModList();
+        }
+
+        Mods.AddRange(_modService.ModList.Where(x => _modService.RecommendedModInstalls.Contains(x.Name)));
+
+        // Make sure that all the recommended mods are selected to start with - this isn't the best way of doing it but it works ¯\_(ツ)_/¯
+        List<ModInfo> selectedMods = [];
+        foreach (ModInfo mod in Mods)
+        {
+            if (_modService.RecommendedModInstalls.Contains(mod.Name))
+            {
+                selectedMods.Add(mod);
+            }
+        }
```

Comment:
I dunno if im sleep deprived but i'm confused by this

Reply:
So am I. I think I was just having a special moment 😓 

Have used something that makes more sense now.